src/commonMain/kotlin/exchange.dydx.abacus/processor/router/skip/SkipProcessor.kt

```diff
@@ -145,7 +145,9 @@ internal class SkipProcessor(
         internalState.tokenResources = tokenResources(selectedChainId)
     }
 
-    private fun getChainById(chainId: String): Map<String, Any>? {
+//    Update this when we move chains to state
+//    https://linear.app/dydx/issue/OTE-470/migrate-chains-and-tokens-to-abacus-state
```

Comment:
It should be easy just to add the `chains` into `PerpetualState`.

Reply:
yeah it shouldn't be too difficult - just thought it made sense logically to do it when i migrate the state to use the new `InternalState` struct

Reply:
synced on slack: fill update this after landing PR to main